src/injected/web/gm-wrapper.js

```diff
@@ -305,15 +302,38 @@ function makeGlobalWrapper(local) {
       return true;
     },
   });
+  const mapWindow = desc => {
+    if (desc && (desc.value === global || desc.value === window)) {
+      desc = {
+        ...desc,
```

Comment:
Babel will be using unsafe calls to Object.xxx methods so it's safer to use `assign()` instead of `...`

Reply:
done